src/modules/pymod/echo.py

```diff
@@ -10,6 +10,6 @@ def echo_cb(h, typemask, message, arg):
 
 def mod_main(h, *args):
     with h.msg_watcher_create(echo_cb, topic_glob="echo.*") as mw:
-        if h.reactor_run(h.get_reactor(), 0) < 0:
+        if h.flux_reactor_run(h.get_reactor(), 0) < 0:
```

Comment:
```suggestion
        if h.flux_reactor_run() < 0:
```

Reply:
Oh, wait. This one won't work. In pymod, we can't use the wrapped `handle.reactor_run()` because signal watchers can't be used in modules. This is because only one ev loop can have a signal registered per process. So we have to fall back to the raw ffi call in the pymod `echo.py`.

Reply:
> Oh, wait. This one won't work. In pymod, we can't use the wrapped handle.reactor_run() because signal watchers can't be used in modules. This is because only one ev loop can have a signal registered per process. So we have to fall back to the raw ffi call in the pymod echo.py.

Perhaps we should add a comment to the file with that info?